### DISCUSSION TITLE
Documentation UI/UX: Footer Component

### DISCUSSION THREAD

**Discussion by philleepflorence** [directus/directus#4328] [Feature Requests]
## What

Add a footer component to both the Documentation Website and Directus Documentation Module.

## Why

Improve Discovery - See [Documentation: A Discovery Summary and Guide.](https://github.com/directus/directus/discussions/4326)  
Show things like last updated as well as what to do if the references, guides are insufficient, incorrect, et al - primarily for new users.

## How

VuePress Footer in Documentation Website or Vue Component in Directus Documentation Module.

## Where

Directus Admin Application - Documentation.
Directus Documentation Website.

**Comment by benhaynes** [comment]
The website already has a footer with an Edit button, and the last modified time. Not sure if this is needed on the internal docs since those are versioned to the installation. As for "what to do if docs are wrong", I think that should be a dedicated section on "Contributing", not text on the bottom of each page.

**Reply by philleepflorence** [reply]
Not sure what the answer is suggesting...there is no need to discuss further?

**Reply by benhaynes** [reply]
I mark them as answered for an initial response, but we can keep discussing the best path forward!

**Reply by philleepflorence** [reply]
Okay good to know!

**Comment by philleepflorence** [comment]
This is an inspiration for the footer component!

![directus-footer-github-inspiration](https://user-images.githubusercontent.com/2633551/109541326-b1658880-7a91-11eb-81ec-3129b53a84dd.png)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
